helm/prometheus-rules/templates/alerting-rules/node-exporter.all.rules.yml

```diff
@@ -13,12 +13,12 @@ spec:
     - alert: NodeExporterCollectorFailed
       annotations:
         description: '{{`NodeExporter Collector {{ $labels.collector }} on {{ $labels.instance }} is failed.`}}'
-      expr: node_scrape_collector_success{collector!~"bonding|hwmon|powersupplyclass|mdadm|nfs|nfsd|tapestats|fibrechannel{{ if eq .Values.managementCluster.provider.kind "azure" }}|nvme{{ end }}"} == 0
+      expr: node_scrape_collector_success{collector!~"bonding|hwmon|powersupplyclass|mdadm|nfs|nfsd|tapestats|fibrechannel|nvme{{ if eq .Values.managementCluster.provider.kind "kvm" }}|pressure{{ end }}"} == 0
```

Comment:
the `conntrack` collector fails too on platforms where `nftables` is used 

* https://github.com/prometheus/procfs/issues/407
* https://github.com/giantswarm/giantswarm/issues/23002#issuecomment-1413364058

It is still worth having the collector running though since it does collect the most basic metrics ( conntrack used and conntrack max ) 

should we add it to the ecxlusion list ?

Reply:
Added to the exclusion list.